.travis.yml

```diff
@@ -30,8 +29,6 @@ matrix:
       env: SYMFONY_VERSION=3.1
     - php: 5.6
       env: SYMFONY_VERSION=3.2
-    - php: 5.3
```

Comment:
please keep this .. but change it to ``5.4`` .. we still want to test the lowest deps